open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/nvd/CvssV2Data.java

```diff
@@ -32,6 +33,7 @@
  * JSON Schema for Common Vulnerability Scoring System version 2.0
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
These do not declare `additionalProperties: false` in schema, so retained.

Reply:
Retained as the schema allows additional properties: https://github.com/jeremylong/Open-Vulnerability-Project/blob/83189a18225029b1cb535f747e6b7662acfd0ecd/open-vulnerability-clients/src/main/resources/json/cvss-v2.0.json